src/khoj/processor/content/github/github_to_entries.py

```diff
@@ -48,7 +48,7 @@ def wait_for_rate_limit_reset(response, func, *args, **kwargs):
         else:
             return
 
-    def process(self, files: dict[str, str] = None, user: KhojUser = None, regenerate: bool = False) -> Tuple[int, int]:
+    def process(self, files: dict[str, str], user: KhojUser, regenerate: bool = False) -> Tuple[int, int]:
```

Comment:
Let's verifying all these functions still work as expected with `--anonymous_mode`. There might be broken assumptions with this change, though I'm not aware of anything off the top of my head.

Reply:
Yeah, tested basic chat, search, agents and automation. Tested anon and auth modes. Don't see any issues yet